src/components/TextInput.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import TextField from 'material-ui/TextField';
+import PropTypes from 'prop-types';
+
+function RenderTextField({ ...props }) {
```

Comment:
This function had to be added to specify exactly how <TextField> should be viewed, which was what Material-UI-Redux was doing for us. 
However i see some issues with passing props from parent to child, When using field for some reason the passed props get two additional properties not needed "Meta", "Input", these props would have to get deleted so that they don't mess up with <Text Field>